src/components/CheckboxList.js

```diff
@@ -0,0 +1,54 @@
+import { Typography } from "@mui/material";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+import DeleteIcon from "@mui/icons-material/Delete";
+import IconButton from "@mui/material/IconButton";
+import Paper from "@mui/material/Paper";
+import Grid from "@mui/material/Grid";
+
+export default function CheckboxList({ addTask, deleteTask }) {
+  const delTask = (id) => deleteTask(id);
+
+  const checkTask = (id) => {
+    addTask.map((task) => {
+      if (task.id === id) {
+        task.iscompleted = !task.iscompleted;
+        console.log(task.iscompleted + " en " + task.id);
```

Comment:
it is recommended to remove `console.log` used for development before to merge this into master.

Reply:
done